client/components/TestRun/index.jsx

```diff
@@ -231,47 +238,69 @@ class TestRun extends Component {
             testsToRun = true;
         }
 
-        let heading, content;
+        let heading = null;
+        let content = null;
+        let iframe = null;
+        let menuUnderIframe = null;
+        let menuRightOfIframe = null;
 
         if (testsToRun) {
             heading = (
-                <React.Fragment>
+                <Fragment>
                     <h2 data-test="test-run-h2">
                         {' '}
                         {`${apg_example_name} (${this.state.currentTestIndex} of ${tests.length})`}
                     </h2>
                     <h3 data-test="test-run-h3">{`${at_name} ${at_version} with ${browser_name} ${browser_version}`}</h3>
                     <h4 data-test="test-run-h4">Testing task: {testName}</h4>
-                </React.Fragment>
+                </Fragment>
             );
+
             if (!this.state.runComplete) {
                 if (!test.result) {
-                    content = (
-                        <React.Fragment>
-                            <iframe
-                                src={`/aria-at/${git_hash}/${
-                                    tests[this.state.currentTestIndex - 1].file
-                                }?at=${at_key}`}
-                                id="test-iframe"
-                                ref={this.testIframe}
-                            ></iframe>
+                    iframe = (
+                        <iframe
+                            src={`/aria-at/${git_hash}/${
+                                tests[this.state.currentTestIndex - 1].file
+                            }?at=${at_key}`}
+                            id="test-iframe"
+                            ref={this.testIframe}
+                        ></iframe>
+                    );
+
+                    menuUnderIframe = (
+                        <ButtonToolbar className="testrun__button-toolbar--margin">
```

Comment:
I wonder if we should give these some kind of landmark region or at least an accessible name to the group.

Make just "region" -- the point being, I want it to be easy for a screen reader user to know those buttons are there and skip to them if partially thru a test..? Regions are all about skipping to area of the web page. I don't know what to label the two different button groups -- or if we make them into "regions", what to call the regions.

Reply:
I'd need to see the generated HTML (or DOM) of the whole page to evaluate this. But a few buttons doesn't necessarily require a landmark.

Reply:
ok ignore this concern @rwaldron